modules/orca/variables.tf

```diff
@@ -82,13 +82,13 @@ variable "lambda_timeout" {
 variable "workflow_config" {}
 
 variable "buckets" {
-  type    = map(object({ name = string, type = string }))
+  type = map(object({ name = string, type = string }))
 }
 
 variable "default_tags" {
-  type = object({team: string, application: string})
+  type = object({ team : string, application : string })
   default = {
-    team: "DR",
-    application: "disaster-recovery"
+    team : "DR",
+    application : "disaster-recovery"
   }
 }
```

Comment:
Add the new optional ORCA variables so they are bubbled up for the user along with type, and description.

    sqs_delay_time 
    maximum_message_size 
    sqs_message_retention_time